src/Radicle/Internal/Parse.hs

```diff
@@ -66,9 +65,12 @@ bracesP = inside "{" "}"
 sqBracketsP :: Parser a -> Parser a
 sqBracketsP = inside "[" "]"
 
+-- Parsing of string literals handles escape sequences just like Haskell does.
 stringLiteralP :: VParser
-stringLiteralP = lexeme $
-    tag =<< StringF . toS <$> (char '"' >> manyTill L.charLiteral (char '"'))
+stringLiteralP = lexeme $ tag =<< StringF . toS <$> escaptedString
+  where
+    escaptedString = catMaybes <$> (char '"' >> manyTill ch (char '"'))
```

Comment:
```suggestion
    escapedString = catMaybes <$> (char '"' >> manyTill ch (char '"'))
```

Reply:
From what I'm seeing in the UI you didn't change the call-site, just the define-site?

Reply:
Wanna make a suggestion on my suggestion?

Reply:
Lol I didn't try